mgmtsystem_nonconformity_hr/views/mgmtsystem_nonconformity_views.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- Copyright 2019 Marcelo Frare (Associazione PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+     Copyright 2019 Stefano Consolaro (Associazione PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl) -->
+     
+<odoo>
+
+    <!-- inherit NC form view 
+            adding department -->
```

Comment:
```suggestion
```
self explainatory ;) no need for comments

Reply:
like before, ok